src/index.html

```diff
@@ -7,8 +7,60 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="#home"
+        class="homeLink"
+      >
+        <img
+          src="images/logo.png"
+          class="image"
+          alt="moyo"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="techList">
+          <li class="listItem">
+            <a href="/" class="link is-active">Apple</a>
+          </li>
+          <li class="listItem">
+            <a href="/" class="link">Samsung</a>
+          </li>
+          <li class="listItem">
+            <a href="/" class="link">Smartphones</a>
+          </li>
+          <li class="listItem">
+            <a href="/"
+              class="link"
+              data-qa="hover"
+            >
```

Comment:
remember about proper formatting, will reject immediately next time if I see this
```suggestion
            <a 
              href="/"
              class="link"
              data-qa="hover"
            >
```